extensions/amp-lightbox/1.0/amp-lightbox.css

```diff
@@ -15,7 +15,7 @@
  */
 
 amp-lightbox {
-  width: '100%';
-  height: '100%';
-  position: 'fixed';
+  width: 100%;
```

Comment:
Just one thing occurred to me. This might break the animations if the lightbox immediately because visible and has a background. We can proceed this way, but need to pay attention to this when we add CSS work.

Reply:
Will do.